Check/SymfonyVersion.php

```diff
@@ -103,14 +103,25 @@ function ($value) use ($branch) {
      */
     private function getResponseAndDecode($url)
     {
-        $opts = [
-            'http' => [
-                'method' => 'GET',
-                'header' => "User-Agent: LiipMonitorBundle\r\n",
-            ],
-        ];
-
-        $array = json_decode(file_get_contents($url, false, stream_context_create($opts)), true);
+        if (class_exists(\Symfony\Component\HttpClient\HttpClient::class)) {
```

Comment:
Can you import this class?

Reply:
sure, I left it in to avoid IDE-confusion when the non-existing class is marked